src/ide/vscode/vscodeIdeUtil.ts

```diff
@@ -0,0 +1,24 @@
+import { TextDocument, TextEditor, TextLine } from "@cursorless/common";
+import * as vscode from "vscode";
+import { VscodeTextDocumentImpl } from "./VscodeTextDocumentImpl";
+import { VscodeTextEditorImpl } from "./VscodeTextEditorImpl";
+import VscodeTextLineImpl from "./VscodeTextLineImpl";
+
+const editorMap = new WeakMap<vscode.TextEditor, TextEditor>();
```

Comment:
I'm skeptical of these module-level variables.  Shall we just add this to the vscode ide object?  Or we could make a `window` / `editorService` that is part of the `ide` 🤷‍♂️

Reply:
moved it into ide